package.json

```diff
@@ -42,14 +46,14 @@
     "eslint-plugin-vue": "^4.3.0",
     "jest": "^22.4.3",
     "jest-serializer-vue": "^1.0.0",
-    "node-sass": "^5.0.0",
-    "poi": "^9.6.13",
+    "poi": "^12.10.3",
```

Comment:
I wonder once we migrate to Vue 3 if we can get rid of `poi` and `rollup-plugin-vue` in favor of `vite` 🤔

Reply:
Yes for sure 🙏  I think I heard they are completely taking this internal once we do Vue 3. I'm kinda expecting this to be a temporary solution for now.